frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -17,12 +18,22 @@ local _ = require("gettext")
 local N_ = _.ngettext
 local T = require("ffi/util").template
 
-local FileSearcher = WidgetContainer:extend{
+local FileSearcher = InputContainer:extend{
     case_sensitive = false,
     include_subfolders = true,
     include_metadata = false,
 }
 
+function FileSearcher:init()
+    self:registerKeyEvents()
+end
+
+function FileSearcher:registerKeyEvents()
+    if Device:hasKeyboard() then
+        self.key_events.ShowFileSearch = { { "Shift", "S" }, args = true }
```

Comment:
```suggestion
        self.key_events.ShowFileSearch = { { "Shift", "F" }, args = true }
```
@poire-z @Frenzie @hius07 changing this to F so I can do the <kbd>Shift</kbd> + <kbd>S</kbd> trick with the last searched term https://github.com/koreader/koreader/pull/12162#issuecomment-2243561999

Reply:
Fine with me.